guides/v2.1/extension-dev-guide/indexing-custom.md

```diff
@@ -58,43 +58,80 @@ All Mview declarations related to a module should be declared in one file.
 
 To push best-selling products to the top of a category listing, process statistics about sales to change the product position dynamically.
 
-Assuming your module is named `<VendorName>_Merchandizing`, you must write the appropriate code in the indexer class:
+Assuming your module is named `<VendorName>_Merchandizing`, you must write the appropriate code in the indexer class `Merchandizing/Model/Indexer/Test.php` 
```

Comment:
```suggestion
Assuming your module is named `<VendorName>_Merchandizing`, you must write the appropriate code in the indexer class `Merchandizing/Model/Indexer/Test.php` 
```
Not should use `Merchandizing/Model/Indexer/Popular.php`
based on the name of the class below:
`class Popular implements \Magento\Framework\Indexer\ActionInterface, \Magento\Framework\Mview\ActionInterface`

Reply:
Yeah. I got it.